src/quantum_systems/composite_quantum_systems.jl

```diff
@@ -132,13 +132,9 @@ struct CompositeQuantumSystem{F1<:Function, F2<:Function} <: AbstractQuantumSyst
         H_drives = sparse.(H_drives)
         G_drives = sparse.(Isomorphisms.G.(H_drives))
 
-        if n_drives == 0
-            H = a -> H_drift
-            G = a -> Isomorphisms.G(H_drift)
-        else
-            H = a -> H_drift + sum(a .* H_drives)
-            G = a -> G_drift + sum(a .* G_drives)
-        end
+        # At least provide one drive
```

Comment:
Can you elaborate a bit on this comment? Not sure I understand the simplification here